spec/controllers/hub/forced_password_resets_controller_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Hub::ForcedPasswordResetsController do
+  let!(:organization) { create :organization, allows_greeters: false }
+  let(:hub_admin_user) { create(:user, role_type: "AdminRole", timezone: "America/Los_Angeles") }
+  let(:non_hub_admin_user) { create(:user, role_type: "GreeterRole", forced_password_reset_at: nil) }
+
+  render_views
+
+  describe "#edit" do
+    context "with a logged in user with non-admin hub access" do
+      before { sign_in non_hub_admin_user }
+
+      it "shows the form for updating the password" do
+        get :edit
+        expect(response).to have_rendered(:edit)
+      end
+
+      it "redirects if the password was already reset" do
```

Comment:
Change this `it` so it doesn't have the word `if` in it by wrapping it in a context block that sets up the scenario. Generally in our tests, we strongly prefer a `context "when ..."` over an `it` with an `if`. The context block gives us a place to change the test setup.